weld/common/src/main/java/org/jboss/as/weld/util/ResourceInjectionUtilities.java

```diff
@@ -50,14 +51,21 @@ public static String getResourceName(String jndiName, String mappedName) {
         }
     }
 
+    /** @deprecated use {@link #getResourceName(InjectionPoint, PropertyReplacer)}  */
+    @Deprecated
```

Comment:
Couldn't we straight up remove this method? It's not any public API and within WFLY I only see a single usage that you already replaced.

Reply:
If you're ok with removing it I'll do it in a separate PR.

I'm cautious about that kind of thing because some layered product might be doing odd things, but really I think using this method would have been unsupported hacking. But I expect this PR will be ported to EAP for bug fix branches so to be super-cautious I'll clean it up in a separate PR.

Reply:
I see what you mean, but I think we could remove it.

Reply:
@manovotn  https://github.com/wildfly/wildfly/pull/13160